src/Core/hcoio_write_esmf_mod.F90

```diff
@@ -151,10 +151,18 @@ SUBROUTINE HCOIO_WRITE_ESMF ( HcoState, RC, OnlyIfFirst, COL )
     ThisDiagn => NULL()
     DO WHILE ( .TRUE. )
 
+       IF (MAPL_am_I_Root()) WRITE(*,*) "Getting next diagnostic in list"
+
        ! Get next diagnostics in list. This will return the next
        ! diagnostics container that contains content to be written
        ! out on this time step.
        CALL Diagn_Get ( HcoState, EOI, ThisDiagn, FLAG, RC, COL=PS )
+
+       IF (MAPL_am_I_Root()) THEN
+          IF ( RC == HCO_SUCCESS .and. FLAG == HCO_SUCCESS ) WRITE(*,*) "Got it! Name: ", TRIM(ThisDiagn%cName)
+          IF ( RC /= HCO_SUCCESS) WRITE(*,*) "Fail! RC: ", RC, "   Flag: ", FLAG
+       ENDIF
```

Comment:
In my GCHP test this is printing lines every timestep.

Reply:
Those wound up being pretty key debug messages. I don't know if we'll need them again, but probably. Can I comment them out but keep them?

Reply:
Yes. We want to overhaul the HEMCO debug prints and if this stays in but commented out it will be included in that update.

Reply:
Okay, done!

Reply:
Thanks!